cmd/wasmd/main.go

```diff
@@ -89,6 +87,7 @@ func newApp(logger log.Logger, db dbm.DB, traceStore io.Writer) abci.Application
 
 	return app.NewWasmApp(
 		logger, db, traceStore, true, invCheckPeriod, skipUpgradeHeights,
+		viper.GetString(flags.FlagHome),
```

Comment:
Yes, this is the same FlagHome (in app.go) so we can just replace homeDir with home.